### ISSUE TITLE
"View latest version" in documentation sometimes links to non-existing page

### ISSUE THREAD

**Issue by LaurentGomila** [SFML/SFML-Website#59]
This refers to the feature added by #58.

The problem is that it links to the same page in the new version. But if the page doesn't exist, this is a dead link.

Example: http://www.sfml-dev.org/documentation/1.6/namespacesf_1_1Style.php

**Comment by eXpl0it3r** [comment]
Well, quite honestly I don't feel like writing some ugly script that would try and figure out what page do and what page don't exist anymore in the latest version...

**Comment by LaurentGomila** [comment]
```
if (exists(page))
    "warning ... click here..."
else
   "warning ..."
```

?

**Comment by mantognini** [comment]
The easiest solution would be to redirect to the main page of the documentation, at least for the fallback.

**Comment by mantognini** [comment]
I quickly implemented that in #65.
